index.js

```diff
@@ -57,8 +57,7 @@ class HyperDHT extends DHT {
       port: opts.port,
       localAddress: encodeAddress(opts.localAddress)
     }
-
-    return this.query('peers', key, opts, cb).map(mapPeers.bind(null, query.localAddress))
+    return this.query('peers', key, query, cb).map(mapPeers.bind(null, query.localAddress))
```

Comment:
very cool 


this was an issue sometimes when you passed through the `opts` object from the parents, and things got references to it, appearing then in the passing reference.